### DISCUSSION TITLE
Accessing your account on another website

### DISCUSSION THREAD

**Discussion by pikachub2005** [jeffalo/ocular#105] [Q&A]
API for changing status/color/starred

**Comment by jeffalo** [comment]
how should this work? it would be bad if any website could update your ocular profile right?

**Reply by pikachub2005** [reply]
I guess... but you already have /react so maybe use something similar?

**Reply by jeffalo** [reply]
i will work on this for ocular-svelte.

**Comment by cobaltt7** [comment]
Maybe some oauth implementation
